lib/solarwinds_apm/opentelemetry/solarwinds_processor.rb

```diff
@@ -51,6 +51,8 @@ def on_finish(span)
 
         return if non_entry_span(span: span)
 
+        SolarWindsAPM::Reporter.send(:report_init) # This only happens if after_fork enabled
```

Comment:
I think i see why you put it here--it's late enough in the tracing pipeline that the reporter connection should be ready, yes? Kind of minor but if the span is not recording we would never get here, and in the future when we switch to OTLP metrics export this bit might get lost... is it possible to move this to the Sampler instead?

Reply:
The init message is meant to be sent on start-up as long as the library is enabled (regardless individual requests sampled or not). When we transition to default OTLP export we will deprecate the init message so it's ultimately not super critical, but in my mind still makes more sense to move to the custom sampler ;)

Reply:
Got it, updated.